ServerCore/Helpers/MailHelper.cs

```diff
@@ -71,17 +71,17 @@ private MailHelper(IConfiguration configuration, bool isDev)
     /// <summary>
     /// Send plaintext mail to a single person.
     /// </summary>
-    public void SendPlaintextOneAddress(string recipient, string subject, string body)
+    public void SendPlaintextOneAddress(string recipient, string subject, string body, string extraBcc = null)
```

Comment:
Consider IEnumerable in case we want to copy groups on mails (e.g. all authors of a puzzle or admins of the event)